linera-execution/src/evm/revm.rs

```diff
@@ -741,8 +748,8 @@ where
 {
     fn instantiate(&mut self, argument: Vec<u8>) -> Result<(), ExecutionError> {
         self.initialize_contract()?;
-        let instantiation_argument = serde_json::from_slice::<Vec<u8>>(&argument)?;
-        if !instantiation_argument.is_empty() {
+        if has_instantiation_function(&self.module) {
```

Comment:
So if this returns `false` - what will happen? How can we `instantiate` a contract (and initialize it - we call `self.initialize_contract`) without it having the `instantiate` entrypoint.